packages/aws_signature_v4/example/pubspec_overrides.yaml

```diff
@@ -0,0 +1,8 @@
+# This file is checked in until it can be easily generated locally
+# and in CI pipelines, e.g. with melos.
+#
+# It may cause issues with publishing until https://github.com/dart-lang/pub/pull/3419
+# is released in stable.
+dependency_overrides:
+  aws_common:
+    path: ../../aws_common
```

Comment:
Does melos not already use the local version?

Reply:
melos does, but mono_repo runs `dart pub upgrade` as its first step, which will break melos

Reply:
Oh I see. It looks like melos is moving toward using this method of overriding dependencies locally and in the future will generate this?

Reply:
Okay. It looks like melos can generate these. Do we want to switch over to that? It appears to be a simple update to melos.yaml. I'm not sure if we would want to then gitignore all the generate files or commit them.

```yaml
command:
  bootstrap:
    usePubspecOverrides: true # pubspec_overrides.yaml will be autogenerate during bootstrap
```

Reply:
Exactly, yep! I tried that and it doesn't work for our repo :/ Still need to investigate more.

https://github.com/aws-amplify/amplify-flutter/pull/1585